packages/access-client/src/agent-data.js

```diff
@@ -6,16 +6,16 @@ import { CID } from 'multiformats'
 import { Access } from '@web3-storage/capabilities'
 import { isExpired } from './delegations.js'
 
-/** @typedef {import('./types').AgentDataModel} AgentDataModel */
+/** @typedef {import('./types.js').AgentDataModel} AgentDataModel */
```

Comment:
This is one of many small changes to ensure all imported module identifiers (e.g. type imports in comments like this) have a file suffix, which is required by `tsc` now that this PR enables `moduleResolution=NodeNext` for all tsc packages.